src/main/scala/carts/cart/events/ItemInsertedIntoCart.scala

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright © 2022-2023 by Pervasive Cats S.r.l.s.
+ *
+ * All Rights Reserved.
+ */
+
+package io.github.pervasivecats
+package carts.cart.events
+
+import io.github.pervasivecats.carts.cart.valueobjects.item.CatalogItem
+import io.github.pervasivecats.carts.cart.valueobjects.item.ItemId
+
+import carts.cart.valueobjects.{CartId, Store}
+
+trait ItemInsertedIntoCart {
+
+  val cartId: CartId
+  val store: Store
+  val catalogItem: CatalogItem
+  val itemId: ItemId
+
```

Comment:
Extra space at the end, missing spaces between vals, move to package "domainevents"